start.py

```diff
@@ -1126,11 +1157,12 @@ def _remove_files(nodes, files):
         node.execute(command=command)
 
 
-def _restart_cm_agents(nodes):
+def _restart_cm_agents(nodes, is_centos_7):
```

Comment:
Is there perhaps a way to check whether there's CentOS 7 in the container directly instead of needing to pass an arg in? That might be a little cleaner than requiring the caller to pass this in.

Reply:
Done.

Reply:
Done.